### PR TITLE
#2142. Change composing inline class tests to be extension type tests

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Inline-classes/composing_inline_classes_A01_t02.dart [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+//                                    ^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+extension type ET3<T1, T2 extends num?>(V1 id) implements V3<T1, T2> {}
+//                                                        ^^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+
+extension type ET4<T1, T2 extends num?>(V1 id) implements V3<T1, T2> {}
```
Comment (eernstg) [inline LanguageFeatures/Inline-classes/composing_inline_classes_A01_t02.dart:43]: Equivalent to line 38?

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t06.dart [hunk 2]
```diff
@@ -2,16 +2,37 @@
...
+/// In particular, if two different declarations of m are inherited from two
+/// superinterfaces then the subinterface can resolve the conflict by
+/// redeclaring m.
+///
+/// There is no notion of having a 'correct override relation' here. With
+/// extension types, any member signature can redeclare any other member
+/// signature with the same name, including the case where a method is
+/// redeclared by a getter, or vice versa.
+///
+/// @description Checks that it is no an error if an extension type declaration
```
Comment (eernstg) [inline LanguageFeatures/Inline-classes/composing_inline_classes_A05_t06.dart:35]:
```suggestion
/// @description Checks that it is not an error if an extension type declaration
```

File: LanguageFeatures/Inline-classes/composing_inline_classes_A05_t06.dart [hunk 3]
```diff
@@ -2,16 +2,37 @@
...
 /// `DV` has superinterface `V` with member `m` and redeclares this member even
```
Comment (eernstg) [inline LanguageFeatures/Inline-classes/composing_inline_classes_A05_t06.dart:36]:
```suggestion
/// `DV` has a superinterface `V` with a member `m`, and redeclares this member even though
```

**Review by eernstg — COMMENTED** [review]
We have a likely update in https://github.com/dart-lang/language/pull/3262, but let's just process this PR using the current rule. (There are a couple of tests that must be adjusted if 3262 is landed as is, but let's look into that when that happens.)

Otherwise, just a couple of typos in this one.

**Comment by sgrekhov** [comment]
Typos fixed

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**